pilot/utils/files.py

```diff
@@ -41,3 +41,17 @@ def create_directory(parent_directory, new_directory):
     os.makedirs(new_directory_path, exist_ok=True)
 
     return new_directory_path
+
+
+def count_lines_of_code(files):
+    total_lines = 0
+
+    for file in files:
+        # Split the content into lines
+        lines = file['content'].split('\n')
+
+        # Count the lines, skipping empty lines and comments if desired
+        line_count = sum(1 for line in lines if line.strip() and not line.strip().startswith('//'))
```

Comment:
I don't think there's a need to skip empty lines and comments. These are usually counted in the file lengths, unless you're talking about "real code lines" which is a bit more complex to calculate than this.

This also only removes one-line JS comments, totally ignoring `#` style comments and multi-line `/* */` comments, so it's not terribly accurate.

Suggested solution:

```
return sum(len(file.splitlines()) for file in files)
```